Error running make on master

Hi,

After cloning the repository and building the code with make as shown in the Readme, I get the following error:

vagrant@ubuntu-xenial:/home/Shared_Folder/liboqs$ make
/bin/mkdir -p include/oqs
cp -f config.h include/oqs
cp -f src/oqs.h include/oqs
cp -f src/common/common.h include/oqs
cp -f src/common/rand.h include/oqs
cp -f src/crypto/aes/aes.h include/oqs
cp -f src/crypto/rand_urandom_aesctr/rand_urandom_aesctr.h include/oqs
cp -f src/crypto/rand_urandom_chacha20/rand_urandom_chacha20.h include/oqs
cp -f src/crypto/sha3/sha3.h include/oqs
cp -f src/kem/kem.h include/oqs
cp -f src/kem/bike/kem_bike.h include/oqs
cp -f src/kem/frodokem/kem_frodokem.h include/oqs
cp -f src/kem/sike/kem_sike.h include/oqs
cp -f src/kex/kex.h include/oqs
cp -f src/kex_ntru/kex_ntru.h include/oqs
cp -f src/kex_rlwe_newhope/kex_rlwe_newhope.h include/oqs
cp -f src/kex_sidh_msr/kex_sidh_msr.h include/oqs
cp -f src/sig/sig.h include/oqs
cp -f src/sig_picnic/sig_picnic.h include/oqs
cp -f src/sig_qtesla/sig_qtesla.h include/oqs
cp -pR -f .libs/liboqs.a
cp: missing destination file operand after '.libs/liboqs.a'
Try 'cp --help' for more information.
Makefile:1144: recipe for target 'links' failed
make: *** [links] Error 1

I've tried different configurations, and different branches, but I always get the same error. I kind of think I'm missing something here, however I just want to use the library with mc_bits enabled, and that didn'T work as well, so I tried the standard build instead. Same problem.

Thanks in advance

--- a/Makefile.am
+++ b/Makefile.am
@@ -117,7 +117,6 @@ endif
 	cp -f src/sig/sig.h include/oqs
 	cp -f src/sig_picnic/sig_picnic.h include/oqs
 	cp -f src/sig_qtesla/sig_qtesla.h include/oqs
-	$(LN_S) -f .libs/liboqs.a
 if ENABLE_MCBITS
 	cp -f src/kex_code_mcbits/kex_code_mcbits.h include/oqs
 endif
@@ -130,7 +129,7 @@ clean-tests:
 	rm -f tests/example_kem tests/speed_kem tests/test_kem tests/minimal_kex_oqs tests/test_kex tests/minimal_sig_oqs tests/test_sig tests/test_rand tests/test_aes tests/test_sha3
 
 clean-kats:
-	rm -rf tests/kat_kem kat_kem_rsp/ 
+	rm -rf tests/kat_kem kat_kem_rsp/
 
 prettyprint:
 	find src tests -name '*.c' -o -name '*.h' | grep -v sig_picnic/external* | grep -v "kex_rlwe_newhope/avx2" | grep -v "kex_sidh_msr" | xargs $(CLANGFORMAT) -style=file -i

